charts/elasticsearch/values.yaml

```diff
@@ -154,6 +170,11 @@ master:
     NODE_INGEST: "false"
     HTTP_ENABLE: "false"
 
+  roles:
+    - master
+    - ml
+    - remote_cluster_client
```

Comment:
there are default roles added from our existing config for master service

Reply:
Yeah, I see those listed in the error message shown in this ticket https://github.com/astronomer/issues/issues/4160